docs/user/connected-accounts.rst

```diff
@@ -103,6 +126,7 @@ Bitbucket
 
 For similar reasons to those above for GitHub, we request permissions for:
 
+* Administering your repositories (to add a read-only deployment key)
```

Comment:
```suggestion
* Administering your repositories (to add webhooks)
```

I see it's similar to GitHub: https://developer.atlassian.com/cloud/bitbucket/bitbucket-cloud-rest-api-scopes/#repository-admin